astropy/extern/ply/cpp.py

```diff
@@ -7,7 +7,6 @@
 #
 # This module implements an ANSI-C style lexical preprocessor for PLY.
 # -----------------------------------------------------------------------------
-from __future__ import generators
```

Comment:
Should we update "extern" packages?

Reply:
I would say not... My understanding is that anything on extern also has to go upstream?

Reply:
I agree!